src/store/slice/user.slice.ts

```diff
@@ -59,20 +59,24 @@ const userSlice = createSlice({
     setUser: (state, { payload }: { payload: { email: string | null } }) => {
       state.isAuth = !!payload.email;
       state.email = payload.email;
+      localStorage.setItem('isAuth', payload.email ? 'true' : 'false');
```

Comment:
It's not correct to write to LS in a reducer, since it's considered a side-effect and reducers have to be pure functions.

Reply:
ok, fixed it